vitest.config.ts

```diff
@@ -6,6 +6,9 @@ export default mergeConfig(
   viteConfig,
   defineConfig({
     test: {
+      deps: {
+        inline: ['@agoric/rpc'],
```

Comment:
What does this do?

Reply:
https://vitest.dev/config/#server-deps-inline
> Vite will process inlined modules. This could be helpful to handle packages that ship .js in ESM format (that Node can't handle).

There were some agoric dependencies which imported file like so
```js
import x from 'foo';
```
but vitest by default would not accept anything other than
```js
import x from 'foo.js';
```

this config fixed that

Reply:
Thanks good to know. I wonder if this is fixed in the latest `@agoric/rpc` versions with https://github.com/Agoric/ui-kit/pull/76, this dapp is using some pretty old versions. But that's a separate issue.